### ISSUE TITLE
can't make last-tool on last06e

### ISSUE THREAD

**Issue by EastEriq** [blumzi/LAST_tools#51]
I've tried to follow https://github.com/blumzi/LAST_tools/wiki/distributing-last-tool and find my way about variations. I am now at:
```
ocs@last06e:~/matlab/LAST/LAST_tools$ git pull
Already up to date.
ocs@last06e:~/matlab/LAST/LAST_tools$ make
/bin/rm -rf packaging
/bin/rm -f last-tool*.deb
test -r files/github-tokens
make: *** [Makefile:101: check-for-github-tokens] Error 1
```
building instead on another machine and copying the .deb, as suggested by the wiki page
```
ocs@last06e:~/matlab/LAST/LAST_tools$ sudo dpkg --install /tmp/last-tool-1.0.9115.28861.deb 
(Reading database ... 267608 files and directories currently installed.)
Preparing to unpack .../last-tool-1.0.9115.28861.deb ...
Unpacking last-tool (1.0.9115.28861) over (1.0.9115.22582) ...
Setting up last-tool (1.0.9115.28861) ...
ocs@last06e:~/matlab/LAST/LAST_tools$ last-tool --version
last-tool: FATAL: macmap_getent_by_mac: Missing mac=74:56:3c:bc:02:ba in /usr/local/share/last-tool/files/MACmap
```

**Comment by agioffe** [comment]
After I copied _github-tokens_ from last01e the build works as it should.
Also, 
> last-tool --version 
is now fine, but I learned that the machine is indeed working through the slower 2.5 Gbit/s (lower) socket, just my mechanism of interface renaming masked it.

**Comment by EastEriq** [comment]
Yeah, it didn't occur to me to check https://github.com/blumzi/LAST_tools/wiki/configfiles#git-tokens, it is obvious that they can't be part of the repo (github will then lock the repo, been there done that).

I reiterate that we should compile a to do list for replacement of computers, like I started about in https://github.com/blumzi/LAST_issues/issues/63

**Comment by agioffe** [comment]
I agree. There is a short list and description of the interface trick in
https://github.com/blumzi/LAST_issues/issues/67

**Comment by blumzi** [comment]
Github-tokens, along with any other data that cannot reside on github, live in the `LAST-CONTAINER`.
The first time we build last-tool on a machine, the tokens need to be fetched from the `LAST-CONTAINER` to the `files `directory.

**Comment by EastEriq** [comment]
Seems that this can be closed, then.
